README.md

```diff
@@ -8,7 +8,7 @@
 
 <img src="./docs/source/common/_static/tt_nn_w_logo.png" alt="ttnn logo" height="150"/>
 
-**TT-NN** is python & C++ Neural Network OP library.
+**TT-NN** is a Python & C++ Neural Network OP library.
```

Comment:
👍 

wonder if we should put this here

TTNN is an open-source machine learning library. It serves as a reference implementation and provides users with necessary tools, examples, and guidelines of how to build on top of tt-metal and to effectively leverage Tenstorrent hardware. 

Also, can we replace TT-NN with TTNN?